### PR TITLE
Remove spatial second object for 700n1a when 35,36,37

### PR DESCRIPTION
**Pull request by dr0i**
The second spatial object is generated from 700n.a for the notations 35, 36,
37. These have to be removed, but only if 700n.b exist.

- update tests

See #1012.

### CODE REVIEW

**Comment by dr0i** [comment]
@acka47 you may want to check the "Files changed" if it's good - I added two test files you mentioned in the issue (HT017536715 and BT000071273).

File: src/test/resources/jsonld/HT017536715 [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      "type" : [ "http://www.wikidata.org/entity/Q897303" ]
+    },
+    "id" : "https://nwbib.de/spatial#Q63078887",
+    "type" : [ "Concept" ],
+    "label" : "Kirchenkreis An der Ruhr",
+    "source" : {
+      "id" : "https://nwbib.de/spatial",
+      "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
+    }
+  } ]
```
Comment (acka47) [inline src/test/resources/jsonld/HT017536715]: The generic N35 "Evangelische Kirche. Kirchenkreise" object is gone now as expected but another object is gone too that should still be there: "Mülheim an der Ruhr" (Q2899), see http://lobid.org/resources/HT017536715.json. From the [source](http://lobid.org/hbz01/HT017536715):

```xml
<datafield tag="700" ind1="n" ind2="1">
  <subfield code="a">99</subfield>
  <subfield code="b">Mülheim an der Ruhr</subfield>
</datafield>
<datafield tag="700" ind1="n" ind2="1">
  <subfield code="a">35</subfield>
  <subfield code="b">Kirchenkreis An der Ruhr</subfield>
</datafield>
```

Comment (dr0i) [reply src/test/resources/jsonld/HT017536715]: Q2899 wasn't yet in the wikidata test's database.
Added it.

**Comment by dr0i** [comment]
Deployed to staging.

**Comment by dr0i** [comment]
Deployed to production, closed.

**Pull request merged**